_data/icons/agora.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "ipfs://QmW3CT4SHmso5dRJdsjR8GL1qmt79HkdAebCn2uNaWXFYh",
```

Comment:
where did you pin this CID - I cannot resolve it

Reply:
@ligi https://gateway.ipfs.io/ipfs/QmW3CT4SHmso5dRJdsjR8GL1qmt79HkdAebCn2uNaWXFYh
Yes, I did the pin again.